src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -586,6 +586,8 @@ private function getArgumentsAsPhp(\DOMElement $node, string $name, string $file
                     $arguments[$key] = new ServiceLocatorArgument($arg);
                     break;
                 case 'tagged':
+                    trigger_deprecation('symfony/dependency-injection', '7.2', 'Type "%s" of tag <%s> in "%s" is deprecated, use "tagged_iterator" instead.', $arg->getAttribute('type'), $name, $file);
```

Comment:
```suggestion
                    trigger_deprecation('symfony/dependency-injection', '7.2', 'Type "tagged" is deprecated for tag <%s>, use "tagged_iterator" instead in "%s".', $name, $file);
```

Reply:
Finding the good wording is always something :smile: I updated both